src/assets/translations/en-us.yaml

```diff
@@ -277,6 +277,14 @@ pathManagement:
       description: 'Rancher Desktop does not change your PATH configuration; add <code>~/.rd/bin</code> to your path manually.'
   accept: Accept
 
+legacyIntegrations:
+  title: Rancher Desktop - Legacy Integrations Found
```

Comment:
I think we could drop the `Rancher Desktop -` if we are using this as the headline of the modal. If we intend this to be the window title, we might need two entries to address each case.